tox.ini

```diff
@@ -1,11 +1,12 @@
 # https://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix
 [tox]
-envlist = py{36,37,38}-graphene{20,21}-pydantic{1}
+envlist = py{36,37,38}-graphene{20,21,30}-pydantic{1}
 isolated_build = true
 
 [testenv]
 whitelist_externals = poetry
 deps =
+    graphene30: graphene>=3.0b5
```

Comment:
Since this PR pins the version of graphene required to 3.0+, it looks like it won't actually be tested against previous versions -- it looks like the API may have breaking changes. If this is desired behavior, can we remove the `graphene20` and `graphene21` test environments, since they're not used?

Reply:
Good catch! I'll remove both test environments.